sql/report_util_functions.sql

```diff
@@ -482,6 +482,35 @@ BEGIN
 END$$ 
 DELIMITER ;
 
+-- patientPickedARVDrugWithoutARVEntrolledDuringReportingPeriod
+
+DROP FUNCTION IF EXISTS patientPickedARVDrugWithoutARVEntrolledDuringReportingPeriod;
```

Comment:
@IipperMangesh I would suggest we make it explicit that the patient doesn't have therapeutic line (i.e `patientPickedARVDrugDuringReportingPeriodWithoutHavingATherapeuticLine`)

Reply:
@IipperMangesh the code from this function is very similar to `patientPickedARVDrugDuringReportingPeriod`, we can avoid code duplication by refactoring patientPickedARVDrugDuringReportingPeriod and get it to call `patientPickedARVDrugDuringReportingPeriodWithoutHavingATherapeuticLine`